.github/.OwlBot.lock.yaml

```diff
@@ -12,6 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 docker:
-  image: gcr.io/cloud-devrel-public-resources/owlbot-nodejs:latest
-  digest: sha256:e92044720ab3cb6984a70b0c6001081204375959ba3599ef6c42dd99a7783a67
-# created: 2023-11-10T00:24:05.581078808Z
+  image: 'gcr.io/cloud-devrel-public-resources/owlbot-nodejs:latest'
+  digest: 'sha256:c7e4968cfc97a204a4b2381f3ecb55cabc40c4cccf88b1ef8bef0d976be87fee'
```

Comment:
not sure if it matters but it doesn't seem like these had quotes beforehand.

Reply:
Good point. I changed it back.